test/distributed/org/apache/cassandra/distributed/test/cql3/SingleNodeTableWalkTest.java

```diff
@@ -88,10 +101,20 @@ protected void preCheck(Cluster cluster, Property.StatefulBuilder builder)
         // CQL_DEBUG_APPLY_OPERATOR = true;
     }
 
-    protected TypeGenBuilder supportedTypes()
+    protected TypeGenBuilder supportedTypes(RandomSource rs)
     {
-        return AbstractTypeGenerators.withoutUnsafeEquality(AbstractTypeGenerators.builder()
-                                                                                  .withTypeKinds(AbstractTypeGenerators.TypeKind.PRIMITIVE));
+        return AbstractTypeGenerators.builder()
+                                     .withTypeKinds(Generators.fromGen(TYPE_KIND_DISTRIBUTION.next(rs)))
+                                     .withPrimitives(Generators.fromGen(PRIMITIVE_DISTRIBUTION.next(rs)))
```

Comment:
before we used uniform random so each test had the same weights.  Now we select a new distribution for each test run (some tests will only be primitives, others will be mostly maps, etc.)